src/services/completions.ts

```diff
@@ -2012,7 +2012,10 @@ namespace ts.Completions {
                 // List of property symbols of base type that are not private and already implemented
                 const baseSymbols = flatMap(getAllSuperTypeNodes(decl), baseTypeNode => {
                     const type = typeChecker.getTypeAtLocation(baseTypeNode);
-                    return type && typeChecker.getPropertiesOfType(classElementModifierFlags & ModifierFlags.Static ? typeChecker.getTypeOfSymbolAtLocation(type.symbol, decl) : type);
+                    return type &&
+                        classElementModifierFlags & ModifierFlags.Static ?

```

Comment:
I think it's a tiny bit more readable to join the predicate into a single line:
```suggestion
                    return type && classElementModifierFlags & ModifierFlags.Static ?
```